statistics.md

```diff
@@ -74,39 +74,77 @@ You can perform full collection using the following syntax.
     {{< copyable "sql" >}}
 
     ```sql
-    ANALYZE TABLE TableNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH|SAMPLES];
+    ANALYZE TABLE TableNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
     ```
 
 + `WITH NUM BUCKETS` specifies the maximum number of buckets in the generated histogram.
 + `WITH NUM TOPN` specifies the maximum number of the generated `TOPN`s.
 + `WITH NUM CMSKETCH DEPTH` specifies the depth of the CM Sketch.
 + `WITH NUM CMSKETCH WIDTH` specifies the width of the CM Sketch.
 + `WITH NUM SAMPLES` specifies the number of samples.
++ `WITH FLOAT_NUM SAMPLERATE` specifies the rate of samples.
+
+`WITH NUM SAMPLES` and `WITH FLOAT_NUM SAMPLERATE` corresponds to two different algorithms of sampling.
+
+- `WITH NUM SAMPLES` specifies the size of the sampling set, which is implemented in the reservoir sampling method in TiDB. When a table is large, it is not recommended to use this method to collect statistics. Because the intermediate result set of the reservoir sampling contains redundant results, it causes additional pressure on resources such as memory.
```

Comment:
```suggestion
- `WITH NUM SAMPLES` specifies the size of the sampling set, which is implemented in the reservoir sampling method in TiDB. When a table is large, it is not recommended to use this method to collect statistics. Because the intermediate result set of the reservoir sampling contains redundant results, and these results cause additional pressure on resources (such as memory).
```